docs/feature_encoders.md

```diff
@@ -26,7 +26,7 @@ If your encoder's clockwise directions are incorrect, you can swap the A & B pad
 #define ENCODER_DIRECTION_FLIP
 ```
 
-Additionally, the resolution can be specified in the same file (the default & suggested is 4):
+Additionally, the resolution can be specified in the same file. The resolution defines how many times the encoder registers a turn before updating its state. The default & suggested is 4, not 1, because many encoders will register multiple turns between each dent. This can be defined with:
```

Comment:
```suggestion
Additionally, the resolution, which defines how many pulses the encoder registers between each detent, can be defined with:
```
Some additional information mostly to help me wrap my own head around this: there are some mentions of the EC11K needing a resolution of 2 in the repo. [This datasheet](https://www.farnell.com/datasheets/1837001.pdf) provides the output waveform from A and B. I'm seeing only two edge transitions happening there.

The encoders that I have, however, are the [Bourns PEC11R](https://www.bourns.com/docs/Product-Datasheets/pec11R.pdf) which I believe are much more commonly used. The waveform for these seems to indicate each detent is towards the end of A's *OFF* pulse, instead of towards the end of *every* pulse as the Farnell datasheet shows, so the default resolution of 4 makes sense here.